main.py

```diff
@@ -3,93 +3,35 @@
-
-
-def labels_by_user_input(**kwargs):
-    data, pull = kwargs["data"], kwargs["pull"]
-    body = data["comment"]["body"]
-    label = "Verified"
-    if "/verified" in body and label not in get_labels(pull=pull):
-        print(f"Adding {label} to {pull.title}")
-        pull.add_to_labels(label)
-
-    if "/unverified" in body:
-        print(f"Removing {label} from {pull.title}")
-        remove_verified_label(pull=pull)
-
-
-def remove_verified_label(pull):
-    label = "Verified"
-    labels = get_labels(pull=pull)
-    if label in labels:
-        pull.remove_from_labels(label)
-
-
-def add_reviewers(**kwargs):
-    data, pull, reviewers = kwargs["data"], kwargs["pull"], kwargs["reviewers"]
-    reviewers = [reviewer.strip() for reviewer in reviewers.split(",")]
-    author = [data["sender"]["login"]]
-    current_reviewers_requests = data["pull_request"]["requested_reviewers"]
-    for reviewer in reviewers:
-        if reviewer not in current_reviewers_requests + author:
-            print(f"Requesting review from {reviewer} for {pull.title}")
-            pull.create_review_request([reviewer])
-
-
-def size_label_prs(**kwargs):
-    data, pull = kwargs["data"], kwargs["pull"]
-    labels = get_labels(pull=pull)
-    additions = data["pull_request"]["additions"]
-    label = None
-    if additions < 20:
-        label = "Size/XS"
-
-    elif additions < 50:
-        label = "Size/S"
-
-    elif additions < 100:
-        label = "Size/M"
-
-    elif additions < 300:
-        label = "Size/L"
-
-    elif additions < 500:
-        label = "Size/XL"
-
-    if label in labels:
-        return
-
-    else:
-        print(f"Labeling {pull.title}: {label}")
-        [pull.remove_from_labels(lb) for lb in labels if lb.startswith("Size/")]
-        pull.add_to_labels(label)
-
-
-ACTIONS = {
-    "labels_by_user_input": labels_by_user_input,
-    "add_reviewers": add_reviewers,
-    "size_label_prs": size_label_prs,
-}
+from src.add_reviewers import add_reviewers
+from src.labels_by_user_input import labels_by_user_input, remove_verified_label
+from src.size_label_prs import size_label_prs
 
 
 if __name__ == "__main__":
     token = os.environ["INPUT_TOKEN"]
     reviewers = os.environ["INPUT_REVIEWERS"]
     action = os.environ["INPUT_ACTION"]
-    with open(os.environ.get("GITHUB_EVENT_PATH"), "r") as fd:
+    event_type = os.environ["GITHUB_EVENT_NAME"]
+    with open(os.environ["GITHUB_EVENT_PATH"], "r") as fd:
         data = json.load(fd)
 
     github = github.Github(token)
     repo = github.get_repo(os.environ["GITHUB_REPOSITORY"])
-    commit = repo.get_commit(os.environ.get("GITHUB_SHA"))
+    commit = repo.get_commit(os.environ["GITHUB_SHA"])
     try:
-        issue_number = data["number"]
+        pull = repo.get_pull(data["number"])
     except KeyError:
-        issue_number = data["issue"]["number"]
+        pull = repo.get_pull(data["issue"]["number"])
+
+    if event_type in ("pull_request_target", "pull_request"):
```

Comment:
we need, this need to be done on pull_request and _target, with your suggestion we will need to call it multiple times

Reply:
That's why you create different workflows that trigger on different events